Sources/Workspace/InitPackage.swift

```diff
@@ -331,13 +411,33 @@ public final class InitPackage {
                 }
             }
             """
+        case .macro:
+            content = """
+            // The Swift Programming Language
+            // https://docs.swift.org/swift-book
```

Comment:
Maybe a more concrete reference to macros once we have it?

Reply:
Yes, makes sense